config/prow/cluster/monitoring/mixins/grafana_dashboards/slo.jsonnet

```diff
@@ -0,0 +1,45 @@
+local config =  import 'config.libsonnet';
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local graphPanel = grafana.graphPanel;
+local prometheus = grafana.prometheus;
+
+
+local dashboardConfig = {
+        uid: config._config.grafanaDashboardIDs['slo.json'],
+    };
+
+dashboard.new(
+        'SLO Compliance Dashboard',
+        time_from='now-7d',
+        schemaVersion=18,
+      )
+.addPanel(
+    graphPanel.new(
```

Comment:
Can we add a [link](https://grafana.com/docs/grafana/latest/linking/panel-links/) to anything that defines what might trigger SLO compliance is? I am not sure what docs we have on failures, or maybe a link to a view we can see failures that were triggered?

Reply:
The time series on the graphs will be labelled with the fields from the relevant alert(s) when not SLO compliant. I think thats should be sufficient for viewing failures.
Once this is demoed I'll add and link some brief docs about how to add alerts to an SLO, how to read the dashboard, how to run more interesting/complex compliance queries, and how to root cause SLO violations. I think the dashboards will be pretty self explanatory and clearly identify the culprit alerts though.